.github/workflows/release.yml

```diff
@@ -0,0 +1,50 @@
+name: Release
+
+env:
+  GH_TOKEN: ${{ github.token }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        type: choice
+        description: "The type of release to make (Minor, Patch, etc)"
+        required: true
+        options:
+          - PATCH
+          - MINOR
```

Comment:
trivial: make it clear when people should pick one or the other

put it in the description like
"Release type (PATCH for bug fixes, MINOR for new features)"

or put it in the option name like:
- bugfix (PATCH)
- new feature (MINOR)